src/main/ruby/core/posix.rb

```diff
@@ -14,9 +14,17 @@ module Truffle::POSIX
     :ushort => :uint16,
     :int => :sint32,
     :uint => :uint32,
+    :long => :sint64,
+    :ulong => :uint64,
   }
 
+  FS_ENCODING = Encoding.find('filesystem')
```

Comment:
@nirvdrum The filesystem encoding is unlikely to change on the fly, right? :)
I don't want to call `Encoding.find('filesystem')` everytime and also it's quite long and verbose.
Maybe we should just have `Encoding::FILESYSTEM` and use that internally?